e2e/app/text-labels.ts

```diff
@@ -55,7 +56,7 @@ export enum MenuOption {
   SuppressCriteriaFromTotalCount = 'Suppress criteria from total count',
   Surveys = 'Surveys',
   Visits = 'Visits',
-  WholeGenomeVariant = 'Whole Genome Sequence'
+  ShortReadWGS = 'Short Read WGS'
```

Comment:
nit: let's keep this alphabetized

Reply:
Good catch. I moved the `LongReadWGS` up and forgot to move this one after renaming